pepper-apis/dsm-core/src/test/java/org/broadinstitute/dsm/TestHelper.java

```diff
@@ -112,7 +112,7 @@ public static void setupDB(boolean setupDDPConfigLookup) {
         cfg = cfg.withValue("quartz.enableJobs", ConfigValueFactory.fromAnyRef("false"));
         cfg = cfg.withValue("portal.port", ConfigValueFactory.fromAnyRef("9999"));
         cfg = cfg.withValue("errorAlert.recipientAddress", ConfigValueFactory.fromAnyRef(""));
-        if (!cfg.getString("portal.environment").startsWith("Local")) {
+        if (!cfg.getString("portal.environment").toLowerCase().startsWith("local")) {
```

Comment:
Just trying to leave the campsite a little nicer than how I found it, since I flubbed "Local" more than once in my config.